docs/modules/statistics.rst

```diff
@@ -31,5 +31,12 @@ Various statistic and metric calculations for analysing inference data.
     :toctree: generated/statistics/
     :template: function.rst
 
-    statistics.weights
-    statistics.utils
+    statistics.utils._broadcast_weights
+    statistics.weights.lat_weight
```

Comment:
For this fix lets just replace the lines here with: `statistics.weights.lat_weight`

Since `_broadcast_weights` was makes as a private function with the underscore in front, lets keep it out of the docs.